apps/new-tab/src/Settings.tsx

```
@@ -585,6 +640,68 @@
+                      onChange={handleFileChange}
+                      id="file-input"
+                      class="block w-full rounded-lg border border-gray-200 text-sm shadow-sm file:me-4 file:border-0 file:bg-gray-50 file:px-4 file:py-3 focus:z-10 focus:border-blue-500 focus:ring-blue-500 disabled:pointer-events-none disabled:opacity-50 dark:border-neutral-700 dark:bg-neutral-900 dark:text-neutral-400 dark:file:bg-neutral-700 dark:file:text-neutral-400"
+                    />
+                  </form>
+                  <br />
+                  {(typeof activePlugins() == "object"
+                    ? activePlugins()
+                    : JSON.parse(activePlugins().toString())
+                  ).map((plugin: PluginItem) => (
+                    <div class="flex w-full items-center justify-between gap-2">
+                      <span class="text-sm">{plugin.fileName}</span>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)